src/java.management/share/classes/javax/management/loading/MLetObjectInputStream.java

```diff
@@ -40,11 +40,13 @@
  */
 class MLetObjectInputStream extends ObjectInputStream {
 
+    @SuppressWarnings("removal")
```

Comment:
Shouldn't `MLetObjectInputStream` be deprecated for removal too? I mean - if MLet was removed - would we need to keep that class? If it were deprecated for removal too then I suspect that there would be no need to suppress the warning here (and below).

Reply:
It's not public so not part of the API, but you may be right that it would reduce the number of places where the warning needs to be suppressed.

Reply:
I tested making the class MLetParser itself @Deprecated in the same way, and it still needs a @SuppressWarnings annotation on the parse(url) method, or it produces 3 warnings regarding use of MLetContent.
I don't mind adding the deprecation note to these two classes to make it clear that these non-public classes are also on the way out.